README.rst

```diff
@@ -2,10 +2,55 @@
 LSST the Docs: Mason
 ####################
 
-**Mason** is the document build tool for LSST software documentation that runs in the Jenkins build environment.
+**Mason** is the build tool for LSST software documentation that runs in the Jenkins build environment.
 It is a part of the **LSST the Docs** (LTD) service for continuous documentation deployment.
 You can learn more about LTD in our `SQR-006`_ technote.
 
+Installation
+============
+
+Mason is intended to work with either Python 2.7, 3.4, or 3.5.
+It *must* run from the same Python as the software being documented, however.
+
+::
+
```

Comment:
I'm pretty sure that any packages provided by EUPS are going to ignore any virtual environment you have set up. EUPS is a virtual environment that lets you switch between versions at will (so is more powerful than a normal virtual environment). What I'm really saying is that the virtual environment is only controlling packages you build with pip and the like. Anything that is under EUPS control will be distinct so setting up the virtual environment before or after lsstsw runs would seem to be irrelevant.